.github/workflows/publish-hotfix.yml

```diff
@@ -83,10 +83,25 @@ jobs:
         run: yarn build
         working-directory: ./canary
 
+  build-test-v2:
```

Comment:
`v2` to differentiate from the existing `build-test` job which builds the apps in `canary`. We'll be combining the two tests as a followup, so the `v2` name is just temporary